terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,15 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# service configs
+use_set_environment_files = true
```

Comment:
Can we set the variable "use_set_environment_files" to true and remove it from here ?